packages/manager/modules/nutanix/src/dashboard/general-info/controller.js

```diff
@@ -22,6 +23,7 @@ export default class NutanixGeneralInfoCtrl {
     this.atInternet = atInternet;
     this.TRAVAUX_LINK =
       TRAVAUX_LINK[coreConfig.getRegion()] || TRAVAUX_LINK.DEFAULT;
+    this.coreConfig = coreConfig;
```

Comment:
```suggestion
    this.NUTANIX_LINK = GUIDE_PACKAGES_URL[this.coreConfig.getUser().ovhSubsidiary] ||
      GUIDE_PACKAGES_URL.DEFAULT;
```
to be compliant with the other links